### ISSUE TITLE
invalid stoi argument - on setValue action in SharePoint List Formatting JSON

### ISSUE THREAD

**Issue by dborchers-gc** [SharePoint/sp-dev-docs#8506]
### Target SharePoint environment

SharePoint Online

### What SharePoint development model, framework, SDK or API is this about?

Declarative list formatting

### Developer environment

Windows

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [X] 💥 Microsoft Edge
- [X] 💥 Google Chrome
- [ ] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Additional environment details

Edge: Version 105.0.1343.53
Chrome: ersion 105.0.5195.127

### Describe the bug / error

If setup an setValue action in my code to change a icon when click on it.

I´ve already done that in another list and it works fine bu´t in this list i got this error
(ive changed the field from Boolean to Text for debugging, so dont wonder, basically i use boolean fields)
![2022-10-15 19_12_32-Microsoft 365 Self-Service-Portal - Einheiten-Administration (EA) - Alle Element](https://user-images.githubusercontent.com/63800439/195999698-ac26fab4-17cd-46cc-be2d-8c20146b7b2d.png)

Thats what the Edge Debugging Tools shows when i click on the Icon:
![2022-10-15 19_14_18-ea_fields xlsx - Excel](https://user-images.githubusercontent.com/63800439/195999719-b0d39904-d30d-49bb-b453-a851533cb8ef.png)

And thats the JSON Code:
![image](https://user-images.githubusercontent.com/63800439/195999742-fff670ba-0316-4b6b-bf13-a4e94a783c34.png)

### Steps to reproduce

This is my first time i got this error, in another List it works fine. It seems to be deeper problem.

### Expected behavior

The Fieldvalue should be cghanged and the other Value should be shown.

**Comment by dborchers-gc** [comment]
Hey @shagra-ms may you have an idea here? Ive got this problems on some lists in the json formatting everytime only when i use the setValue action

**Comment by fowl2** [comment]
It may be unrelated, but I've seen this error can come from the offline list sync function.

This page might help: [Common issues syncing Microsoft SharePoint lists offline](https://learn.microsoft.com/en-us/sharepoint/troubleshoot/lists-and-libraries/common-sync-issues)

**Comment by shagra-ms** [comment]
@fowl2 you are right. This is caused by offline list sync function. We are working with the concerned team to fix the issue.

**Comment by shagra-ms** [comment]
@dborchers-gc as a workaround, you can stop syncing the concerned list. Please use the steps shared [here](https://support.microsoft.com/en-us/office/edit-lists-offline-41403c3e-1795-4e07-b56b-ae591cbde2f9).

**Comment by dborchers-gc** [comment]
Thanks for the Tips i will test that
